config/application.rb

```diff
@@ -27,7 +27,7 @@ class Application < Rails::Application
       g.helper false
     end
 
-    config.action_controller.default_protect_from_forgery = false
+    config.action_controller.default_protect_from_forgery = true
```

Comment:
I think this is [the default setting](https://guides.rubyonrails.org/configuring.html#config-action-controller-default-protect-from-forgery), so I don't think you need this unless your intention is to be the most explicit possible.

Reply:
If it's turned to "false" in Phase 1, then I figure it's better to leave it and change to "true" in Phase 2. Yes, to be explicit.